app/models.py

```diff
@@ -364,11 +364,24 @@ class AuditEvent(db.Model):
         object_type, object_id
     )
 
+    acknowledged = db.Column(
+        db.Boolean,
+        index=True,
+        unique=False,
+        nullable=False)
+
+    acknowledged_by = db.Column(db.String)
+    acknowledged_at = db.Column(
+        db.DateTime,
+        nullable=True,
+        default=datetime.utcnow)
```

Comment:
Providing a default here will set the `acknowledged_at` time when the record is created.